driver/protocol.md

```diff
@@ -34,14 +34,19 @@ Server-driver communication uses **protobuf 3** for message encoding. In the
 driver side, this is implemented in the UAST normalizer in Go, using the
 babelfish SDK.
 
-* **TODO: Add proto**
+You can see the messages used for the **protobuf 3** exchange on the [proto file
```

Comment:
We are currently using JSON representation for server-driver communication as produced by Go's `encoding/json` package.